packages/plugin-doc-coverage/src/lib/config.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
+
+export const docCoveragePluginConfigSchema = z
+  .object({
+    skipAudits: z.array(z.string()).optional(),
```

Comment:
You could add a description field to the model to make it more discoverable

Reply:
Ops forgot about this one and the next one, omw